src/Pages/Result/Result.css

```diff
@@ -0,0 +1,91 @@
+.result-outer-container {
```

Comment:
`result_container-outer`

Reply:
can we use two special character at a time or we have to follow one only to maintain consistency.
Here I used "-" for consistency.

Reply:
Here `result_container` is common in both the css Code block so better to approach using BEM

Reply:
okey